guardrails/src/adapters/repositories/postgres_guardrail_repository.py

```
@@ -0,0 +1,419 @@
+                    version.name,
+                    version.revision,
+                    version.is_active,
+                    version.created_at,
+                    version.description,
+                )
+
+                # Materialize the version
+                await self._materialize(conn, version.id)
+
+                return version
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)